src/components/trace/hat-script-graph.ts

```diff
@@ -445,14 +440,17 @@ class HatScriptGraph extends LitElement {
         </hat-graph>
         <div class="actions">
           <mwc-icon-button
-            .disabled=${paths.length === 0 || paths[0] === this.selected}
+            .disabled=${Object.keys(this.getTrackedNodes()).length === 0 ||
+            Object.keys(this.getTrackedNodes())[0] === this.selected}
```

Comment:
Why calculate the keys so often? Can we just calculate it above the return? 

Alternative is `this.getTrackedNodes()[0] === this.selected` since calling an unknown index returns undefined.

Reply:
I have no idea why, but while I previously could not get this to work, I now cannot get it to break...

Reverted this change.